packages/db/src/config.ts

```diff
@@ -105,6 +109,40 @@ export const getConnectionConfig = async ({
   return await getConnectionConfigEnv(env);
 };
 
+/**
+ * Check if a string can be converted to a number
+ *
+ * @param bigIntValue - string to be converted to number
+ * @returns true if the string can be converted
+ * @throws - Throws error if the string can not be converted
+ */
+export const safelyConvertBigInt = (bigIntValue: string): boolean => {
```

Comment:
minor comment - the name of the function makes it seem as if it will do the conversion, I think `canSafelyConvertBigInt` or something like that might be more explicit about what the function is doing